decidim-elections/config/locales/en.yml

```diff
@@ -37,6 +37,10 @@ en:
               is_taken: is taken
             public_key:
               cant_be_changed: can't be changed
+        trustees_participatory_space:
+          attributes:
+            base:
+              is_taken: Name is taken
```

Comment:
💬 it would be nice to customize a bit this message, with something like: "There is already another Trustee using a username like yours and it has to be unique. Please, change it in your account and try it again."